auth-api/migrations/alembic.ini

```diff
@@ -45,6 +45,11 @@ args = (sys.stderr,)
 level = NOTSET
 formatter = generic
 
+[logger_auth_api]
+level = DEBUG
```

Comment:
We want `auth_api` logging in the migration logs, it's much easier and cleaner than using `print`